extensions/prototype-content-with-issues-table/app.R

```diff
@@ -31,34 +32,7 @@ get_failed_job_data <- function(item, usage) {
     {
       get_jobs(item) |> 
         # filter successful jobs
-        filter(exit_code != 0) |> 
-        # map content job types to something more readable 
-        mutate(tag = case_when(
-               tag %in% c("build_report", "build_site", "build_jupyter") ~ "Building",
-               tag %in% c("packrat_restore", "python_restore") ~ "Restoring environment",
-               tag == "configure_report" ~ "Configuring report",
-               tag %in% c("run_app", 
-                          "run_api", 
-                          "run_tensorflow", 
-                          "run_python_api",
-                          "run_dash_app",
-                          "run_gradio_app",
-                          "run_streamlit",
-                          "run_bokeh_app",
-                          "run_fastapi_app",
-                          "run_voila_app",
-                          "run_pyshiny_app") ~ "Running",
-               tag == "render_shiny" ~ "Rendering",
-               tag == "ctrl_extraction" ~ "Extracting parameters",
-               TRUE ~ tag)) |>
-        # map exit codes to something more readable 
-        mutate(exit_code = as.character(exit_code)) |>
-        mutate(exit_code = case_when(
-               exit_code %in% c("1", "2", "134") ~ "failed to run / error during running",
-               exit_code == "137" ~ "out of memory",
-               exit_code %in% c("255", "15", "130") ~ "process terminated by server",
-               exit_code %in% c("13", "127") ~ "configuration / permissions error",
-               TRUE ~ exit_code))
```

Comment:
The above was moved to the table render code in response to:

> [What do you think about moving this munging into the display code rather than the querying code?](https://github.com/posit-dev/connect-extensions/pull/33#discussion_r1989910956)

Reply:
Thanks!